sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -290,6 +289,32 @@ private[ui] class ExecutionPagedTable(
         }
       }
 
+    var headerNameWithTooltips: Map[String, String] = Map("ID" -> "Execution ID of the SQL query.",
+      "Description" ->
+        ("Description of the SQL query. When click on the description link," +
+          " it will navigate to query details page."),
+      "Submitted" -> "Submitted",
+      "Duration" ->
+        "Difference between start time and close time.")
```

Comment:
Could you try like the following?
```scala
    var headerNameWithTooltips = Map(
      "ID" -> "Execution ID of the SQL query.",
      "Description" -> ("Description of the SQL query. When click on the description link," +
        " it will navigate to query details page."),
      "Submitted" -> "Submitted",
      "Duration" -> "Difference between start time and close time.")
```

Reply:
ok i will change like above